modules/add.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] As the previous code reviewer mentioned, please fix the linters and remove the `eslint-disable` lines from the Javascript files.